x/ccv/provider/keeper/keeper.go

```diff
@@ -398,63 +402,54 @@ func (k Keeper) GetAllUnbondingOpIndexes(ctx sdk.Context, chainID string) (index
 	defer iterator.Close()
 
 	for ; iterator.Valid(); iterator.Next() {
-		// parse key to get the current VSC ID
-		_, vscID, err := types.ParseUnbondingOpIndexKey(iterator.Key())
-		if err != nil {
-			panic(fmt.Errorf("failed to parse UnbondingOpIndexKey: %w", err))
-		}
-
-		var index ccv.UnbondingOpsIndex
-		if err = index.Unmarshal(iterator.Value()); err != nil {
-			panic("Failed to unmarshal JSON")
+		var vscUnbondingOps types.VscUnbondingOps
+		if err := vscUnbondingOps.Unmarshal(iterator.Value()); err != nil {
+			panic(fmt.Errorf("failed to unmarshal VscUnbondingOps: %w", err))
 		}
 
 		indexes = append(indexes, types.VscUnbondingOps{
-			VscId:          vscID,
-			UnbondingOpIds: index.GetIds(),
+			VscId:          vscUnbondingOps.GetVscId(),
+			UnbondingOpIds: vscUnbondingOps.GetUnbondingOpIds(),
 		})
 	}
 
 	return indexes
 }
 
-// GetUnbondingOpIndex gets an unbonding index,
-// i.e., the IDs of unbonding operations that are waiting for
+// GetUnbondingOpIndex gets the IDs of unbonding operations that are waiting for
 // a VSCMaturedPacket with vscID from a consumer with chainID
-func (k Keeper) GetUnbondingOpIndex(ctx sdk.Context, chainID string, valsetUpdateID uint64) ([]uint64, bool) {
+func (k Keeper) GetUnbondingOpIndex(ctx sdk.Context, chainID string, vscID uint64) ([]uint64, bool) {
 	store := ctx.KVStore(k.storeKey)
 
-	bz := store.Get(types.UnbondingOpIndexKey(chainID, valsetUpdateID))
+	bz := store.Get(types.UnbondingOpIndexKey(chainID, vscID))
 	if bz == nil {
 		return []uint64{}, false
 	}
 
-	var idx ccv.UnbondingOpsIndex
-	if err := idx.Unmarshal(bz); err != nil {
-		panic("Failed to unmarshal UnbondingOpsIndex")
+	var vscUnbondingOps types.VscUnbondingOps
+	if err := vscUnbondingOps.Unmarshal(bz); err != nil {
+		panic(fmt.Errorf("failed to unmarshal VscUnbondingOps: %w", err))
 	}
 
-	return idx.GetIds(), true
+	return vscUnbondingOps.GetUnbondingOpIds(), true
 }
 
-// DeleteUnbondingOpIndex deletes an unbonding index,
-// i.e., the IDs of unbonding operations that are waiting for
+// DeleteUnbondingOpIndex deletes the IDs of unbonding operations that are waiting for
 // a VSCMaturedPacket with vscID from a consumer with chainID
-func (k Keeper) DeleteUnbondingOpIndex(ctx sdk.Context, chainID string, valsetUpdateID uint64) {
+func (k Keeper) DeleteUnbondingOpIndex(ctx sdk.Context, chainID string, vscID uint64) {
```

Comment:
Not going to ask you to take this out obviously, but this kind of thing shouldn't have been in this PR. We already have an issue in the backlog to go over all the naming

Reply:
True. I need to control my OCD :)